irods/api_number.py

```diff
@@ -179,4 +179,5 @@
     "REPLICA_CLOSE_APN": 20004,
     "TOUCH_APN": 20007,
     "AUTH_PLUG_REQ_AN": 1201,
+    "AUTHENTICATION_APN": 110000,
```

Comment:
I guess it's too late to do anything about it now, but there seems to be an inconsistent use of `AN` vs `APN` suffixes in this list.

Reply:
Ohhh, I see. These are based on those. Carry on.